src/Network/Probe.php

```diff
@@ -1853,11 +1853,17 @@ private static function ensureAbsoluteLinkFromHTMLDoc(string $href, string $base
 	 */
 	private static function feed(string $url, bool $probe = true): array
 	{
-		$curlResult = DI::httpClient()->get($url, HttpClientAccept::FEED_XML);
+		try {
+			$curlResult = DI::httpClient()->get($url, HttpClientAccept::FEED_XML);
+		} catch(\Throwable $e) {
+			return [];
```

Comment:
Can you add some logging here? I would like to find the origin of these invalid urls.

Reply:
This specific method is provided WebFinger addresses when earlier detection fails. Since it's fed user-provided content, I'm not sure logging would help.

Reply:
Still, I just added some logging.

Reply:
I think that we should fail at the most early stage. Logging helps here.

Reply:
I'm inclined to think you're right, after all I got this error because the custom WebFinger implementation in APContact was incomplete and the `feed()` method was given a WebFinger address.